ui/src/kapacitor/containers/TickscriptPage.tsx

```diff
@@ -316,7 +316,7 @@ export class TickscriptPage extends PureComponent<Props, State> {
       }
     } catch (error) {
       console.error(error)
-      notify(notifyTickscriptLoggingError(error))
+      notify(notifyTickscriptLoggingError('Could not collect kapacitor logs'))
```

Comment:
i think we have a central constants file for all error copy

Reply:
this was previously just passing through an error and there was nothing in the notifications copy, happy to move.